docker/data-science/notebooks/ch07/2.semantic-search.ipynb

```diff
@@ -825,7 +792,7 @@
        "    'type_ss': ['Restaurants, Korean, Thai'],\n",
        "    'latitude_d': 35.171873,\n",
        "    'longitude_d': -80.849032,\n",
-       "    '_version_': 1790795163517321228},\n",
+       "    '_version_': 1794601456732471304},\n",
```

Comment:
It's weird that _version_ is coming back unless we're doing a "*" in the "fl" parameter. I'm also noticing that all the fields are using dynamic schema (name_t, city_t, type_ss, etc.).

It probably makes more sense to explicitly define the schema for these fields and use their clean names (type, latitude, longitude, name, city, etc.) and then only return the fields we need instead of "*" fields. We'll have to do this for other engines anyway, so this will make the Solr code more consistent.

Listing 7.1 is the relevant place where the reviews collection is being created and dynamic schema is currently generated.

Reply:
Added a task for explicitly defining schema for collections used by the SKG